packages/plugin-session-replay-browser/package.json

```diff
@@ -46,9 +46,13 @@
     "tslib": "^2.4.1"
   },
   "devDependencies": {
+    "@amplitude/analytics-browser": ">=2 <3",
```

Comment:
It won't for end customers, since they will not install this package when they install the plugin (that's why theres a difference b/t dependencies and devDependencies). I haven't observed any issues when developing locally with this, but let's observe if there are any!